specification/servicenetworking/resource-manager/Microsoft.ServiceNetworking/preview/2022-10-01-preview/TrafficController.json

```diff
@@ -1008,7 +1008,7 @@
     "AssociationType": {
       "type": "string",
       "enum": [
-        "subnets"
+        "Subnet"
```

Comment:
I thought we agreed to use the plural `"Subnets"` for `AssociationType`? NFVRP is already [serializing as plural](https://msazure.visualstudio.com/One/_git/Networking-nfv?path=/src/NfvRp/Core/Frontend/Models/TrafficController/Enums/AssociationType.cs&version=GBdevelop2&line=18&lineEnd=18&lineStartColumn=9&lineEndColumn=16&lineStyle=plain&_a=contents).